src/interfaces/IPoolManager.sol

```diff
@@ -180,11 +177,11 @@ interface IPoolManager is IFees, IClaims {
     /// @dev Can also be used as a mechanism for _free_ flash loans
     function take(Currency currency, address to, uint256 amount) external;
 
-    /// @notice Called by the user to move value into Claims balance
-    function mint(Currency token, address to, uint256 amount) external;
+    /// @notice Called by the user to move value into ERC6909 balance
+    function mint(address to, uint256 id, uint256 amount) external;
 
-    /// @notice Called by the user to redeem their Claims balance
-    function burn(Currency token, uint256 amount) external;
+    /// @notice Called by the user to move value from ERC6909 balance
+    function burn(address from, uint256 id, uint256 amount) external;
```

Comment:
Hmm what about placing these in the IERC6909Claims interface? Just thinking now that especially we've changed the interface to match the underlying 6909 one maybe it should? 

Could go either way tho..

Reply:
Eh I think they should be in PoolManager since they are wrappers around the internal _mint and _burn and since they have the delta accounting logic in them